codesy/base/management/commands/check_issue_status.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+
+from django.core.management.base import BaseCommand
+
+from auctions.models import Issue
+from gh_gql import Issue as ghIssue
+
+logging.basicConfig()
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+
+        issues = Issue.objects.all()
```

Comment:
remove this and just do `for issue in Issue.objects.all():` until we need the QuerySet in another place.